src/frontend/playerState/index.ts

```diff
@@ -28,21 +28,21 @@ export interface ExpectedQueryParameters {
 }
 
 /**
- * Type of editor, either `instant` for an instant remix or `full` for the code editor.
+ * Type of editor, either `instant` to customize the Koji app's configuration or `full` to edit the app's code in the Koji code editor.
```

Comment:
Both values are sent when someone is "customizing" that app. `instant` means in the player and `full` means in the code editor. The code editor way is really not a thing anymore, so we might ask Sean if he wants to remove this and related properties from a future version of the package/platform.  For now, maybe something like...
```suggestion
 * Type of editor the creator is using, either `instant` for the Koji player or `full` for the Koji code editor.
```

Reply:
How about we just say, "Must be set to `instant`." until Sean removes this option?

Using the code editor means they have to fork the project, so the "creator" becomes a "developer" at that point, and it's an entirely different scenario then.

Reply:
This value is not *set*. It is returned to a callback method so the developer could use it to tailor the experience based on the editor type. So, best to describe the possible values until/unless we remove them.

Reply:
Replaced.